### PR TITLE
Fix title edit icon persistence (#311)

### PR DESCRIPTION
**Pull request by nimishamukherjee**
Fix title edit icon persistence (#311)

### CODE REVIEW

**Comment by nimishamukherjee** [comment]
@ldimaggi could you have a look at the functional tests for this PR? I made some changes to them.
The UI behaviour change: 
On the details pane, the edit icon was visible next to the work item title. As per #311 we need to remove this edit icon and display it only on hover.

**Comment by ldimaggi** [comment]
The test changes are fine - we will write a separate test to verify that the hovering is successful - thanks!

File: src/tests/work-item/work-item-list/work-item-detail.page.js [hunk 1]
```diff
@@ -80,9 +80,35 @@ WorkItemDetailPage.prototype  = Object.create({}, {
   },
 
   /* Icon edit buttons */
-  workItemTitleEditIcon:  {
+  editHover:   {
     get: function ()
-    { return element(by.id("workItemTitle_btn_edit")); }
+    { 
+      return element(by.id(""));
```
Comment (joshuawilson) [inline src/tests/work-item/work-item-list/work-item-detail.page.js]: Does this ID need to reference something? (Or maybe I just don't understand how it works.)

**Comment by joshuawilson** [comment]
@nimishamukherjee please rebase

**Pull request merged**